client/lib/abtest/index.js

```diff
@@ -88,14 +91,15 @@ ABTest.prototype.init = function( name ) {
 		throw new Error( 'A default variation is specified for ' + name + ' but it is not part of the variations' );
 	}
 
-	const languageSlugs = map( config( 'languages' ), 'langSlug' );
-
-	if (
-		testConfig.localeTargets &&
-		! every( testConfig.localeTargets, ( target ) => {
-			return languageSlugs.indexOf( target ) !== -1;
-		} ) ) {
-		throw new Error( 'localeTargets array contains invalid language slugs.' );
+	this.localeTargets = [ 'en' ];
+	if ( testConfig.localeTargets ) {
+		if ( 'any' === testConfig.localeTargets ) {
+			this.localeTargets = false;
+		} else if ( isArray( testConfig.localeTargets ) && every( testConfig.localeTargets, langSlugIsValid ) ) {
```

Comment:
Just to note: `every( testConfig.localeTargets, langSlugIsValid )` -- I do like that much more. 👍